### PR TITLE
Fix Conditional Statements: Add nodes to capture comments

### PR DESCRIPTION
**Pull request by Adarsh-NP**
This PR fixes the issue when the if cleanup rules were not working due to comments inside the blocks. Added comment nodes as optional to cover those scenarios.

### CODE REVIEW

File: src/cleanup_rules/swift/rules.toml
Comment (ketkarameya) [inline src/cleanup_rules/swift/rules.toml:253]: how about just `(_)*`

File: test-resources/swift/cleanup_rules/input/SampleClass.swift
Comment (ketkarameya) [inline test-resources/swift/cleanup_rules/input/SampleClass.swift:222]:
```suggestion
            // to be preserved 1
```

File: test-resources/swift/cleanup_rules/input/SampleClass.swift
Comment (ketkarameya) [inline test-resources/swift/cleanup_rules/input/SampleClass.swift:230]:
```suggestion
            // to be preserved 2
```

similarly for others for the sake of consistency

**Review by ketkarameya — COMMENTED** [review]
Do u really need an alternation between comment and statement? Would wildcard not work ?

File: src/cleanup_rules/swift/rules.toml
Comment (Adarsh-NP) [reply src/cleanup_rules/swift/rules.toml:253]: We want to capture the first (statements) block separately. If we do just (_)* we will catch all the if else cases since else and else if are also considered as children of the first if.

Comment (ketkarameya) [reply src/cleanup_rules/swift/rules.toml:253]: But then this does not seem a sound rule either. How abut altering the grammar for swift to fix these `if-else` awkwardness from the root ?

Comment (Adarsh-NP) [reply src/cleanup_rules/swift/rules.toml:253]: Waiting for that sounds a little unreasonable to me. Also, we would need to confirm with @Satyam1749 because if I remember he wanted to pick that up but it was an optional improvement.

Comment (ketkarameya) [reply src/cleanup_rules/swift/rules.toml:253]: Its alright if you dont want to take this up, we will not land this either . When I get time i will look into it

Comment (Adarsh-NP) [reply src/cleanup_rules/swift/rules.toml:253]: How about we merge this while we look to solve that? Once solved, we can revert this change because if we solve grammar a lot of rules will have to be re-written anyway. Especially these if else cleanups.

Comment (Satyam1749) [reply src/cleanup_rules/swift/rules.toml:253]: @ketkarameya `(comments)* .(statements)` is a full proof way to handle if-blocks with current grammar and there can't be an edge case to it. So, in my opinion we should go ahead with this for now. Since, fixing the tree-sitter grammar is a 3rd party dependency. 

This usecase is a pretty basic usecase so we shouldn't keep this blocked on 3rd party dependency.

**Review by ketkarameya — APPROVED** [review]

**Pull request merged**
